<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logIn</class>
 <widget class="QMainWindow" name="logIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>logIn</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: silver;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>350</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>350</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/data/mainbg.jpg);</string>
          </property>
          <property name="title">
           <string>WELCOME TO OKUYU MANAGEMENT</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>70</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:blue;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string> USERNAME:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>120</y>
             <width>91</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:blue;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>PASSWORD:</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonLogin">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>230</y>
             <width>141</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:green;</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonCancel">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>230</y>
             <width>141</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:red;</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEditUsername">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>70</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:blue;</string>
           </property>
           <property name="placeholderText">
            <string>Enter your username</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEditPassword">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>120</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:blue;</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Enter your password</string>
           </property>
          </widget>
          <widget class="QLabel" name="labelImage">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>171</width>
             <height>171</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/data/user-avatar.png);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
